### DISCUSSION TITLE
http://localhost:8080/v2/config/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#485] [Announcements]
# http://localhost:8080/v2/config/

Config Intro View Details (intro.md); I18n Support View Details (i18n.md); Theme Config Theme Basic Config (theme/basic.md); Theme Feature Config (theme/feature.md); Theme Layou...

http://localhost:8080/v2/config/

**Comment by yuanhuajin** [comment]
# Test Comment

**Comment by yuanhuajin** [comment]
# TEST COMMENT
